tour_ui/config/tour.tour.backdrop.json

```diff
@@ -0,0 +1,206 @@
+{
+    "_config_name": "tour.tour.backdrop",
+    "title": "Backdrop",
+    "name": "backdrop",
+    "editable": 1,
+    "paths": [
+        "<front>"
+    ],
+    "description": "This is a beginners tour of Backdrop CMS. ",
+    "auto_start": 1,
+    "enabled": 1,
+    "steps": {
+        "Welcome": {
+            "title": "Welcome",
+            "text": "Welcome to this tour of your new Backdrop CMS site. We will start here on the front page and show you some of the most important things you need to know about your site. ",
+            "selector": ".block-hero",
```

Comment:
Keep in mind, this will find the first hero block on the page, if there are more than one. Also if a user had removed the hero and then installed the module, this would be a JS error.